CoWorkSpace/Spaces.Common/Services/SpaceService.cs

```diff
@@ -28,17 +28,17 @@ public Task<IEnumerable<Space>> GetAllAsync()
 
         public Task<Space> GetByIdAsync(string Id)
         {
-            return this.repository.GetSpaceById(Id);
+            return this.repository.GetSpaceByIdAsync(Id);
         }
 
-        public Task InsertAsync(Space space)
+        public Task AddAsync(CreationInfo creationInfo)
         {
-            return this.repository.InsertSpace(space);
+            return this.repository.AddSpaceAsync(creationInfo);
         }
 
-        public Task<bool> UpdateAsync(Space space)
+        public Task<bool> UpdateAsync(CreationInfo creationInfo)
         {
-            return this.repository.UpdateSpace(space);
+            return this.repository.UpdateSpaceAsync(creationInfo);
```

Comment:
Da li si sigurna da ovo radi? Ovaj servis treba da primi space i da ga prosledi repo-u. Sa druge strane, vidim da repo prima Space a ti mu prosledjujes CreationInfo. Proveri to